### PR TITLE
Add jupyter-collaboration

### PR DESCRIPTION
**Pull request by martinRenou**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ x] Source is from official source.
- [ x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x ] If static libraries are linked in, the license of the static library is packaged.
- [x ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ x] Build number is 0.
- [x ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/jupyter-collaboration/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  license_family: BSD
+  license_file: LICENSE
+  dev_url: https://github.com/jupyterlab/jupyter_collaboration
+
+extra:
+  recipe-maintainers:
+    - hbcarlos
+    - martinRenou
+    - fcollonval
+    - davidbrochart
```
Comment (martinRenou) [inline recipes/jupyter-collaboration/meta.yaml:53]: cc. @hbcarlos @davidbrochart @fcollonval Please add a thumb up if you agree to maintain!

Comment (bollwyvl) [reply recipes/jupyter-collaboration/meta.yaml:54]: another tack would be to add `@conda-forge/jupyterlab`, which can be helpful when coordinated (breaking) releases go out, and timing gets tricky

File: recipes/jupyter-collaboration/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    - jupyter_events
+    - jupyter_server_fileid >=0.6.0,<1
+
+test:
+  imports:
+    - jupyter_collaboration
+  requires:
+    - pip
+  commands:
+    - pip check
```
Comment (bollwyvl) [inline recipes/jupyter-collaboration/meta.yaml:39]: recommend verifying _at least_ e.g. `labextension list`, `serverextension list`: this has caught real problems on other projects that switched up their `build_system`, etc. But really would recommend running at least _some_ of the test suite.

File: recipes/jupyter-collaboration/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+about:
+  home: https://github.com/jupyterlab/jupyter_collaboration
+  summary: 'A Jupyter extension providing support for collaborative features in JupyterLab'
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  dev_url: https://github.com/jupyterlab/jupyter_collaboration
```
Comment (bollwyvl) [inline recipes/jupyter-collaboration/meta.yaml:47]: Would add:

`docs_url: https://jupyterlab-realtime-collaboration.readthedocs.io`

(that's a mouthful)

Comment (bollwyvl) [reply recipes/jupyter-collaboration/meta.yaml:47]: Also might be worth a more extension, multi-line `description`.

File: recipes/jupyter-collaboration/meta.yaml (hunk 1 above)
Comment (bollwyvl) [reply recipes/jupyter-collaboration/meta.yaml:63]: sry, it's without `@`, a la https://github.com/conda-forge/ypy-websocket-feedstock/blob/main/recipe/meta.yaml#L54 (it gets rendered with the `@` in readme)

Comment (martinRenou) [reply recipes/jupyter-collaboration/meta.yaml:63]: Thanks!

**Comment by martinRenou** [comment]
I'm disabling the tests on windows for now. There is a notebook_shim issue that is unrelated.

**Pull request merged**